platform/security/idp/security-idp-client/src/test/java/org/codice/ddf/security/idp/client/LogoutRequestServiceTest.java

```diff
@@ -93,24 +105,8 @@
 
   private SecurityTokenHolder securityTokenHolder;
 
-  private class MockLogoutRequestService extends LogoutRequestService {
-
-    public MockLogoutRequestService(
-        SimpleSign simpleSign, IdpMetadata idpMetadata, RelayStates<String> relayStates) {
-      super(simpleSign, idpMetadata, relayStates);
-    }
-
-    @Override
-    protected void buildAndValidateSaml(
-        String samlRequest,
-        String relayState,
-        String signatureAlgorithm,
-        String signature,
-        SignableXMLObject xmlObject)
-        throws ValidationException {
-      return;
-    }
-  }
```

Comment:
The test was creating  a new subclass of the class-under-test. It did to disable the LogoutRequestService's validation routine. I do not like mocking the class under test, so I removed it.  I update the tests so they passed validation.